app/models/energy/schedulebase.py

```diff
@@ -43,3 +25,28 @@ class ScheduleUnitType (str, Enum):
     percent = 'Percent'
     control = 'Control'
     mode = 'Mode'
+
+
+class ScheduleType(BaseModel):
+    """Specifies the data types and limits for values contained in schedules."""
+
+    type: Enum('ScheduleType', {'type': 'ScheduleType'})
+
+    name: str = Schema(
+        ...,
+        regex=r'^[\s.A-Za-z0-9_-]*$',
+    )
+
+    lower_limit_value: float = Schema(
+        0,
+        description='Lower limit for the schedule type is entered.'
+    )
+
+    upper_limit_value: float = Schema(
+        1,
+        description='Upper limit for the schedule type is entered.'
+    )
+
+    numeric_type: ScheduleNumericType
```

Comment:
Let's make the default here `Continuous`, which is what I think the E+ default is.